### DISCUSSION TITLE
"Failed to mount SharedSupport.dmg"

### DISCUSSION THREAD

**Discussion by telepathine-ltd** [grahampugh/erase-install#173] [Q&A]
Hi all, 

We cached the 11.6.1 installer in our environment using a simple curl command, and have been running the script to some success. However, we're seeing failures at about a two-to-one ratio. One error is "Error obtaining valid installer", and the other reads 

 [check_installer_is_valid] Mounting SharedSupport.dmg failed
   [check_installer_is_valid] Build of existing installer could not be found!

Anyone have any insight as to why this might happen?

**Comment by grahampugh** [comment]
@telepathine-ltd Where did you get the installer from? Is it a full installer or a stub installer?

**Reply by leifdavisson** [reply]
I had this issue to on some of my more locked down macs. 

Script result: 
   [erase-install] v26.0 script execution started: Thu May 26 14:08:29 PDT 2022
   [erase-install] Caffeinating this script (pid=10164)
   [check_free_space] OK - 941 GB free/purgeable disk space detected
   [check_power_status] OK - AC power detected
   [erase-install] Looking for existing installer app or pkg
   [find_existing_installer] Installer found at /Applications/Install macOS Monterey.app.
   [check_installer_is_valid] Checking validity of /Applications/Install macOS Monterey.app.
   [check_installer_is_valid] Mounting SharedSupport.dmg failed
   [check_installer_is_valid] Build of existing installer could not be found!

   [erase-install] attempting to terminate the 'caffeinate' process - Termination message indicates success
/Library/Management/erase-install/erase-install.sh: line 1042: 10176 Terminated: 15          /usr/bin/caffeinate -dimsu -w $$

**Reply by leifdavisson** [reply]
Here is my command line.

/Library/Management/erase-install/erase-install.sh --update --reinstall --check-power --user --confirm --overwrite

**Reply by grahampugh** [reply]
You cannot block mounting of disk images.
